packages/toolkit/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@reduxjs/toolkit",
-  "version": "1.9.2-11",
+  "version": "1.9.1",
```

Comment:
:hand: Not sure why this has been changed - any changes here should be reverted

Reply:
test